### DISCUSSION TITLE
Can I use one custom subdomain to access my user- and project-pages?

### DISCUSSION THREAD

**Discussion by drandarov-io** [community/community#42477] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

- I have setup my user page at https://gh.dmitr.io/
- I have also set up a project page at https://drandarov-io.github.io/embed-life-time-progressbars/
- The CNAME record is "Name: gh" and "Value: drandarov-io.github.io"

Is it possible to access the project page at https://gh.dmitr.io/embed-life-time-progressbars/ and what would I need to configure? Currently this link does not work.

**Comment by yoannchaudet** [comment]
Hello,

The behavior you are describing is supported.

Your problem is that gh.dmitr.io is not configured on your user site, but on another project site. You need the repository to be called: `drandarov-io.github.io` instead of `drandarov-io`. Renaming the repository should be sufficient.

**Comment by drandarov-io** [comment]
![image](https://user-images.githubusercontent.com/13102251/209388342-5efe3868-9f0b-4b03-a96e-30755db567b4.png)



I didn't realize that this was a project site. It makes sense now. Thanks. It's now working as expected.
